### PR TITLE
add sentinel-web-servlet-6x module and support springboot3.0. issue:#…

### PR DESCRIPTION
**Pull request by LiYangSir**
…2998

### Describe what this PR does / why we need it
Add sentinel-web-servlet-6x module, support springboot3.0

### Does this pull request fix one issue?
Fixes: #2998 

### Describe how you did it
Replace javax.servlet-api dependency with jakarta.servlet-api. The package name uses jakarta to replace javax

### Describe how to verify it
environment：springboot:3.0.0, JDK17
1. create a new FilterBean and set the Filter to CommonFilter. CommonTotalFilter is the same
```java
    @Bean
    public FilterRegistrationBean myFilter() {
        FilterRegistrationBean myFilter = new FilterRegistrationBean();
        myFilter.addUrlPatterns("/*");
        myFilter.setFilter(new CommonFilter());
        return myFilter;
    }
```
2. create a HelloController to test
```java
    @GetMapping("/hello")
    public String hello() {
        try (Entry entry = SphU.entry("hello")) {
            return "hello";
        } catch (BlockException e) {
            return "world";
        }
    }
```
3. Check whether the cluster point link detects the cluster point and whether the corresponding rule takes effect
image
image
image

### Special notes for reviews
None

### CODE REVIEW

File: sentinel-adapter/sentinel-web-servlet-6x/pom.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <dependencies>
+        <dependency>
+            <groupId>com.alibaba.csp</groupId>
+            <artifactId>sentinel-core</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>jakarta.servlet</groupId>
+            <artifactId>jakarta.servlet-api</artifactId>
+            <version>5.0.0</version>
```
Comment (drgnchan) [inline sentinel-adapter/sentinel-web-servlet-6x/pom.xml:28]:
```suggestion
            <version>${servlet.api.version}</version>
```

File: sentinel-adapter/sentinel-web-servlet-6x/pom.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        <groupId>com.alibaba.csp</groupId>
+        <artifactId>sentinel-adapter</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>sentinel-web-servlet-6x</artifactId>
+    <packaging>jar</packaging>
+
+    <properties>
+        <servlet.api.version>5.0.0</servlet.api.version>
```
Comment (drgnchan) [inline sentinel-adapter/sentinel-web-servlet-6x/pom.xml:16]: should the version be 6.0.0?

Comment (LiYangSir) [reply sentinel-adapter/sentinel-web-servlet-6x/pom.xml]: Modified

**Comment by liwu96** [comment]
为啥老哥关闭了这个呀？

**Comment by zhangdc007** [comment]
怎么close了，最新的1.8.8 还是不支持jakarta 啊，CommonFilter() 的file 还是javax的包，无法升级到spring boot 3.x

**Pull request closed without merging**